snippets/Visual Studio/ruiiv4.snippet

```diff
@@ -20,18 +20,18 @@
       </Declarations>
       <Code Language="csharp">
         <![CDATA[public static readonly DependencyProperty ViewModelProperty =
-            DependencyProperty.Register("ViewModel", typeof($classname$Model), typeof($classname$), new PropertyMetadata(null));
+            DependencyProperty.Register(nameof(ViewModel), typeof($classname$Model), typeof($classname$), new PropertyMetadata(null));
```

Comment:
I seen people do `PropertyMetadata(typeof($classname$Model)))` instead of `PropertyMetadata(null))`

This way it handles value types.

Reply:
Thanks for the info.